core/src/main/java/org/hisp/dhis/android/core/category/CategoryCategoryComboLinkStoreImpl.java

```diff
@@ -46,14 +64,48 @@ public long insert(CategoryCategoryComboLinkModel entity) {
         return executeInsert();
     }
 
+    @Override
+    public int delete() {
+        return databaseAdapter.delete(CategoryCategoryComboLinkModel.TABLE);
+    }
+
+    @Override
+    public boolean delete(@NonNull CategoryCategoryComboLinkModel entity) {
```

Comment:
In all stores in this method that delete by uid returns a int type, please can you refactor it?

Reply:
I changed the return type to an int but this entity is a link table and it has two primary keys if I remove the object I will have to have two uids instead of one object.

Do you have that I use two uids instead of one object?